File: Emlak/src/app/admin/model/IGaleriDil.ts
export interface IGaleriDil {
    ID: number,
    //[Required(ErrorMessage = "Baðlý Galeri alaný boþ olamaz.")]
    GalID: number,
    //[Required(ErrorMessage = "Dil alaný boþ olamaz.")]
    TransID: number,
    //[Required(ErrorMessage = "Galeri Adý alaný boþ olamaz ve en fazla 255 karakter olmalýdýr.")]
    //[StringLength(255)]
    GalleryName: string,
    ShortText1: string,
    ShortText2: string,
    //[DataType(DataType.MultilineText)]
    //[AllowHtml]
    Description: string,
    Deleted: boolean,

    Mesaj: string,

    GalleryList: any[],
    TranslationList: any[]
}

File: AdminPanel/Repository/GaleriDil/IGaleriDil.cs
using System.Collections.Generic;
using System.Web.Mvc;

namespace Repository.GaleriDilModel
{
    public interface IGaleriDil
    {
        int ID { get; set; }
        int GalID { get; set; }
        int TransID { get; set; }
        string GalleryName { get; set; }
        string ShortText1 { get; set; }
        string ShortText2 { get; set; }
        string Description { get; set; }
        bool Deleted { get; set; }

        string Mesaj { get; set; }

        List<SelectListItem> GalleryList { get; set; }
        List<SelectListItem> TranslationList { get; set; }
        string GalleryAdi { get; set; }
        string TranslationAdi { get; set; }

        List<GaleriDil> List();
        IGaleriDil Select(int id);
        bool Insert(IGaleriDil galeri);
        bool Update(IGaleriDil galeri);
        bool Delete(int id);
        bool Remove(int id);
    }
}
